rules/windows/process_access/sysmon_in_memory_assembly_execution.yml

```diff
@@ -22,9 +22,12 @@ logsource:
     product: windows
 detection:
     selection1: 
-        CallTrace: 
-            - "C:\\Windows\\SYSTEM32\\ntdll.dll+*|C:\\Windows\\System32\\KERNELBASE.dll+*|UNKNOWN(*)"
-            - "*UNKNOWN(*)|UNKNOWN(*)"
+        - CallTrace|contains|all: 
+              - 'C:\\Windows\\SYSTEM32\\ntdll.dll+'
+              - '|C:\\Windows\\System32\\KERNELBASE.dll+'
+              - '|UNKNOWN(*)'
+        - CallTrace|endswith:
+              - "UNKNOWN(*)|UNKNOWN(*)"
```

Comment:
the asterisks here are masks, so it's better to reconstruct the rule in a way to get rid of them (: 
(the same goes for line 28)

Reply:
Any suggestion on:
```
        - CallTrace|endswith:
              - 'UNKNOWN(*)|UNKNOWN(*)'
```
To me, it seems necessary

Reply:
here you go:

```
detection:
    selection1:
        CallTrace|contains|all:
              - 'C:\\Windows\\SYSTEM32\\ntdll.dll+'
              - '|C:\\Windows\\System32\\KERNELBASE.dll+'
              - '|UNKNOWN('
              - ')'
    selection2:
        CallTrace|contains|all:
              - "UNKNOWN("
              - ")|UNKNOWN("
        CallTrace|endswith: ")"
    selection3: 
        CallTrace|contains: "UNKNOWN"
    granted_access:
        GrantedAccess:
            - "0x1F0FFF"
            - "0x1F1FFF"
            - "0x143A"
            - "0x1410"
            - "0x1010"
            - "0x1F2FFF"
            - "0x1F3FFF"
            - "0x1FFFFF"
    condition: (selection1 OR selection2) or (selection3 AND granted_access)
```

Reply:
fixed